person-service/src/test/java/com/github/saintukrainian/cloudcrud/controllers/AbstractTest.java

```diff
@@ -21,14 +20,13 @@
 public abstract class AbstractTest {
 
   protected MockMvc mvc;
-  protected static final String PERSONS_URL = "http://localhost:8080/persons/";
-  protected static final String PWD_URL = "http://localhost:8080/pwd/";
-  protected static final String PD_URL = "http://localhost:8080/pd/";
+  protected static final String PERSONS_URL = "http://person-service/persons/";
+  protected static final String PWD_URL = "http://person-service/pwd/";
```

Comment:
Just as suggestion for future - use more informative names, for example persons-with-details.